source/deploy/postgres-migration.rst

```diff
@@ -109,6 +109,29 @@ To drop indexes, run the following commands before the migration (These are incl
    DROP INDEX IF EXISTS idx_posts_message_txt;
    DROP INDEX IF EXISTS idx_fileinfo_content_txt;
 
+Artifacts may remaining from previous configurations/versions
+~~~~~~~~~~~~~~~~~
+
+Prior to ``v6.4``, Mattermost was utilizing `golang-migrate <https://github.com/golang-migrate/migrate>`__ for handling the schema migrations. Since we don't use it anymore, the table ``schema_migrations`` shouldn't be migrated. If you were using Mattermost before ``v6.4`` consider excluding this table from migration and comparison. 
+
+.. code:: sql
+
+   DROP TABLE mattermost.schema_migrations;
+
+Another note about configurations is that; if you were previously utilizing a database for handling the Mattermot configuration, those tables should be removed from your MySQL database. Consider running following DDL to drop tables.
```

Comment:
```suggestion
Another note about configurations is that; if you were previously utilizing a database for handling the Mattermost configuration, those tables should be removed from your MySQL database. Consider running following DDL to drop tables.
```

Reply:
Is our target audience for this content knowledgeable about what ``DDL`` stands for?

Reply:
I'd say yes if they are executing these queries :) But wouldn't hurt to add extra info I suppose, shall I add?

Reply:
I recommend expanding out the acronym once, the first time it's referenced on the page, followed by the acronym itself in brackets. You could then use the acronym instead, and readers will know what it stands for.